frame/support/src/lib.rs

```diff
@@ -428,6 +428,13 @@ macro_rules! parameter_types_impl_thread_local {
 					pub fn set(t: $type) {
 						[<$name:snake:upper>].with(|v| *v.borrow_mut() = t);
 					}
+
+					/// Mutate the internal value in place.
+					pub fn mutate<F: FnOnce(&mut $type) -> ()>(mutate: F) {
```

Comment:
This is a unrelated change -- small enough to sneak in here, but I can also extract.